netstandard/ref/System.IO.Enumeration.cs

```diff
@@ -0,0 +1,52 @@
+namespace System.IO.Enumeration
+{
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential, Size = 1)]
+    public ref partial struct FileSystemEntry
```

Comment:
Why is it not readonly?

Reply:
The structure is mutable. It lazy initializes itself as you call different methods.